lstmcpipe/scripts/useful_tmp_scripts/rerun_irfs_mem_issue.py

```diff
@@ -0,0 +1,36 @@
+"""
+Run python rerun_irfs_mem_issue.py lstmcpipe_config.yaml
+"""
+from ruamel.yaml import YAML
+import sys
+from pathlib import Path
+
+
+yaml = YAML()
+
+filename = sys.argv[1]
+
+cfg = yaml.load(open(filename).read())
+
+cfg['stages_to_run'] = ['dl2_to_irfs']
+cfg['stages'].pop('r0_to_dl1')
```

Comment:
You could add a default value to get in case the key does not exists. Otherwise looks fine to me